ui-v2/tests/lib/page-object/visitable.js

```diff
@@ -44,9 +44,22 @@ export function visitable(path, encoder = encodeURIComponent) {
       let executionContext = getExecutionContext(this);
 
       return executionContext.runAsync(context => {
-        let params = assign({}, dynamicSegmentsAndQueryParams);
-        let fullPath = fillInDynamicSegments(path, params, encoder);
-
+        var params;
+        let fullPath = (function _try(paths) {
+          const path = paths.shift();
+          params = assign({}, dynamicSegmentsAndQueryParams);
+          var fullPath;
+          try {
+            fullPath = fillInDynamicSegments(path, params, encoder);
+          } catch (e) {
+            if (paths.length > 0) {
+              fullPath = _try(paths);
+            } else {
+              throw e;
+            }
+          }
+          return fullPath;
+        })(typeof path === 'string' ? [path] : path.slice(0));
```

Comment:
Some comments in this file might be nice - just uncertain why this is being overridden and what the differences are between it and the ember-page-object one.

Reply:
good call 👍 This is growing because the 3rd party one fits my needs less and less, might call it something else also to be clearer that it's not the normal one.